src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,8 @@
 Release History
 ===============
 
+* configure kubectl bash-completion during "az aks install-cli"
```

Comment:
Because this change doesn't modify the acs command_module, these release notes shouldn't be modified. Sorry if our PR template was misleading here!